concepts/packages/links.json

```diff
@@ -1,35 +1,10 @@
 [
   {
-    "url": "https://golang.org/ref/spec#Assignments",
-    "description": "assignment"
+    "url": "https://golang.org/ref/spec#Import_declarations",
+    "description": "import-declaration"
```

Comment:
The descriptions can be a little more meaningful like "import-declaration-spec"

Is the spec really the best links we can give them? I suspect there are good links to "A Tour of Go" which would be better: https://tour.golang.org/